docs/development/setup/mermaid.md

```
@@ -0,0 +1,35 @@
+# mermaid-cli use
+
+## CLI cmds 
+
+**install**
+```bash
+npm install -g @mermaid-js/mermaid-cli
+```
+
+**usage**
```

markdownlint-cli2 (0.17.2)

Emphasis used instead of a heading
null

(MD036, no-emphasis-as-heading)